index.html

```diff
@@ -843,6 +843,64 @@ <h2>Skills</h2>
    
     </section>
     <!--about me section ends here -->
+     
+              
+    <!--contact me   -->
+    <section class="contact">
+
+      <div class="contact-container">
+
+          <div class="contact-container-desktop">
+           
+            <h2 id="contact">Contact me</h2>
+            
+            <p class="contact-P contact-P-desktop">
+              If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+              I’d love to help with it!
+            </p>
+
+              <!--  contact form section  -->
+              <form class="contact-form" action="https://formspree.io/f/xaygnnvv" method="POST">
+                
+                <ul class="form-section">
+
+                  <li>
```

Comment:
👍 You've done a great job improving your UI based on the previous reviewer's feedback! However, I noticed that you have a margin of 32px, whereas the design specifies it should be 24px. To ensure your UI aligns with the design requirements, please update your code and set the margin to 24px as indicated in the design.
![image](https://github.com/melkamu372/Portfolio-mobile-first/assets/55840999/b75279ad-d7c2-4552-8db1-dc5c04ccef3c)

By making this adjustment, your UI will be consistent with the design and maintain a visually pleasing layout. Keep up the good work, and I'm confident your attention to design details will result in a polished final product! 🎨🚀

Reply:
@shahadat3669  please clarify where you get 32px margin my code is 
![image](https://github.com/melkamu372/Portfolio-mobile-first/assets/47281626/68787525-29b3-4bf9-8a64-3502ba44d762)